reproducible/ubuntu/test/BUILD

```diff
@@ -11,7 +11,7 @@ structure_test(
 structure_test(
     name = "ubuntu_build_test",
     config = "//tests:ubuntu_build_test.yaml",
-    image = "//reproducible/ubuntu:ubuntu_build",
+    image = "//reproducible/ubuntu:ubuntu_16_0_4_build",
```

Comment:
Looks like you missed the rename up on line 5:

```shell
ERROR: /home/travis/gopath/src/github.com/GoogleCloudPlatform/debian-docker/reproducible/ubuntu/test/BUILD:5:1: no such target '//reproducible/ubuntu:ubuntu': target 'ubuntu' not declared in package 'reproducible/ubuntu' defined by /home/travis/gopath/src/github.com/GoogleCloudPlatform/debian-docker/reproducible/ubuntu/BUILD and referenced by '//reproducible/ubuntu/test:ubuntu_test'
```

Reply:
fixed